src/main/java/uk/gov/hmcts/reform/pip/rules/rules/RulesService.java

```diff
@@ -33,4 +42,87 @@ public Publication getPublication(Integer publicationId) {
             throw new PublicationNotFoundException(String.format("Publication with ID %s not found", publicationId));
         }
     }
+
+    /**
+     * Method to get a court.
+     * @param courtId The ID of the publication to get.
+     * @return The court itself, if it passes the business rules and is found.
+     */
+    public CourtHearings getCourt(Integer courtId) {
```

Comment:
are you sure this is a rule? this seems very out of the ordinary being here

Reply:
sorry I am not sure what you mean

Reply:
none of these are business rules, as far as i know we dont have any business rules ready so im not sure who told you this was one but can you put the 3 amigos that led you to this either on this chat or somewhere we can see

Reply:
There was not 3 amigos. How I see this is a proof of concept that proof the link between FE and BE and we can refactor in the future development once the higher design will be approved.

Reply:
this is a huge story and theres a lot of decisions that looks like have been made, we cant assume any of these and would need approval for how youve made up rules and used this as a POC

however this comment isnt necessarily relating to it being a POC, whether or not it is a concept we still shouldnt add rules that arent rules, this looks like its logic, `getCourt` is not a rule, its a function